src/components/Dropdown.tsx

```diff
@@ -38,7 +53,7 @@ export default function Dropdown({
 
     if (value) {
       const index = currentValues.indexOf(value);
-      if (e.metaKey) {
+      if (e.metaKey || e.ctrlKey) {
```

Comment:
what's this do? no objection per se  just wanted to understand. don't think this works on mac but maybe it does on windows or linux?

Reply:
"Meta" key is equal to what the rest of the world knows as a "Control" key. :)